api/app.py

```diff
@@ -16,7 +15,13 @@ def index():
 def landing():
     emissionsAndLandData = PreprocessData.emissionsAndLand()
     emissionAndCerealYieldData = PreprocessData.emissionAndCerealYield()
-    populationAndArableLand = PreprocessData.populationAndArableLand()
```

Comment:
Based on the below mentioned change requests, new way to call should be as such:

```
# instantiate the class
preprocessor = PreprocessData()
# call the function
populationAndArableLand = preprocessor.populationAndArableLand()
```